srcpkgs/nextdns/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'nextdns'
+pkgname=nextdns
+version=1.37.11
+revision=1
+build_style=go
+go_import_path="github.com/nextdns/nextdns"
+short_desc="NextDNS CLI client (DoH Proxy)"
+maintainer="RunningDroid <runningdroid@zoho.com>"
+license="MIT"
+homepage="https://nextdns.io/"
+distfiles="https://github.com/${pkgname}/${pkgname}/archive/refs/tags/v${version}.tar.gz"
+checksum=f70cb424b0ae47456c8579d3910b486f75afde34572f28f8cf6eb1222e59b88b
+
+post_patch() {
+	vsed -i main.go -e "s/version  = \"dev\"/version  = \"${version}\"/"
```

Comment:
this should probably be done with go ld flags (unless that doesn't work?)

Reply:
I wasn't aware I could do that, thanks for the tip!